build.sh

```diff
@@ -20,6 +20,18 @@ check_for python
 check_for svn
 check_for unzip
 check_for xml2-config
+check_for bash
+check_for java
+check_for javac
+check_for xzdec
+
+case `uname` in
+    OpenBSD)
+        check_for egcc
+        check_for eg++
+	;;
+esac
```

Comment:
I now agree, as Debian has GCC 4.9.2 and OpenBSD 4.9.3. But that's not related to this PR as such.